mod/embed/views/default/elgg/embed.js

```diff
@@ -0,0 +1,189 @@
+/**
```

Comment:
I have just copied over the old library, changed `elgg.embed` references to `embed`. Moved deprecated insert out of here, and added the `ready` flag, as I was seeing duplicate bindings earlier